codespell_lib/data/dictionary.txt

```diff
@@ -1795,6 +1826,17 @@ arugments->arguments
 arund->around
 asbtraction->abstraction
 asceding->ascending
+asend->ascend
```

Comment:
also "as end"

Reply:
is that even grammatically correct?

Reply:
Maybe a random example: "The inclusion of 31 December 2006 as end date [...]"

Reply:
Added